e2e/cypress/tests/integration/enterprise/cloud/billing/cloud_pricing_modal_spec.js

```diff
@@ -336,7 +336,7 @@ describe('Pricing modal', () => {
         cy.get('.CloudUsageModal').contains('Cloud Starter limits');
     });
 
-    it('should open team downgrade selection modal when attempting downgrading from paid to starter', () => {
+    it('should open team downgrade selection modal when attempting downgrading from enterprise trial to starter', () => {
```

Comment:
Hey @neallred requesting you take a look at this test case. Maybe I am missing something